data/demo-modules.yml

```diff
@@ -50,3 +50,7 @@ delta:
   depends_on: bravo
   draft: true
   summative_threshold: 70
+test:
```

Comment:
Is this getting used? I'd rather we keep the mock content format.

Reply:
I did not use it in my test, but all the existing tests that used it failed if it was not there